dissect/target/plugins/os/unix/locale.py

```diff
@@ -46,9 +47,7 @@ def language(self):
             if (path := self.target.fs.path(locale_path)).exists():
                 for line in path.open("rt"):
                     if "LANG=" in line:
-                        found_languages.append(
-                            line.replace("LANG=", "").replace('"', "").replace(".UTF-8", "").replace("_", "-").strip()
-                        )
+                        found_languages.append(normalize_language(line.replace("LANG=", "").strip()))
 
         return found_languages
```

Comment:
Can't put a comment there, but no standardization necessary for the `timezone` function here I assume?

Reply:
Correct, Unix systems generally already return IANA tzdb values.